README.md

```diff
@@ -27,3 +27,36 @@ SimpleUI is a easy UI Framework for building and generating web user interfaces.
 - `npm run langium:generate && npm run build` to generate the infrastructure and build the project
 
 To run booth watches use `npm run watch && npm run langium:watch`.
```

Comment:
Just a simple typo fix: replace booth with both.
also as far as I know it's "watchers" not "watches" but I might be wrong there

Reply:
Yeah, your right!